gradle-plugins/model-compiler/build.gradle

```diff
@@ -101,13 +103,15 @@ protobuf {
  */
 task prepareProtocConfig(type: Copy) {
     description = "Prepares the spine_protoc.gradle file"
-    
+
     from "./spine_protoc.gradle"
     into "./src/main/resources"
 
-    filter(org.apache.tools.ant.filters.ReplaceTokens, tokens: ['baseVersion': "'$project.spineVersion'".toString(),
-                                                                'pbVersion': "'$project.protobufVersion'".toString(),
-                                                                'gRPCVersion': "'$project.gRpcVersion'".toString()])
+    filter(ReplaceTokens, tokens: ['baseVersion': "'$project.spineVersion'".toString(),
+                                   'pbVersion': "'$project.protobufVersion'".toString(),
+                                   'gRPCVersion': "'$project.gRpcVersion'".toString(),
+                                   'spineDir': "'.spine'".toString()])
+    // TODO:2017-08-31:dmytro.dashenkov: Replace ".spine" with "Extension.SPINE_BUILD_ARTIFACT_STORAGE_DIR".
```

Comment:
Not sure why this TODO is here. Can you at least reference an issue?

Reply:
Referenced (below).